### DISCUSSION TITLE
Discussion: Lots of Private Branches

### DISCUSSION THREAD

**Discussion by michael-hawker** [microsoft/microsoft-ui-xaml#7483] [General]
# Discussion: Private Branches
We had an issue on the Windows Community Toolkit about the number of non-active/private branches we had on the main repo: https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/3321

Looks like WinUI is getting more branches too, which makes it a bit harder to realize there's release branches for each release.

Contributors should instead fork the repo and create branches in their forks and then create pull requests from there.

I've been considering turning off write access for branches on the main repo to ensure contributors use forks as well...

Just wanted to carry this suggestion forward.

**Comment by StephenLPeters** [comment]
@ranjeshj and @jevansaks FYI

**Comment by jevansaks** [comment]
The PR policy to delete branches after merge helped a lot. But there's still quite a lot of stale non-working branches. Perhaps people who don't clean up their stale branches get their repo write permissions revoked? 😲

**Comment by kmgallahan** [comment]
@anawishnoff

ಠ_ಠ

![Screenshot 2020-12-13 150736](https://user-images.githubusercontent.com/5379488/102022460-f8769400-3d54-11eb-8a57-22dd3891d59a.jpg)

**Comment by anawishnoff** [comment]
@kmgallahan just cleaned some of mine up :)
